app/diversity/get.controller.js

```diff
@@ -0,0 +1,24 @@
+const { getSentencePlanComments } = require('../../common/data/sentencePlanningApi')
+const { getCommentText } = require('../../common/utils/getCommentText')
+
+const getDiversity = async (
+  { path, errors, errorSummary, body, renderInfo, params: { planid }, session: { 'x-auth-token': token } },
+  res
+) => {
+  let renderDetails = renderInfo
```

Comment:
Two options here:
In the original de-structure you can set a default `renderInfo = {}`. You'll still have to re-assign to another `let` though due to linting rules not allowing you to modify the function param though.
Alternatively you can assign a default with an or: `let renderDetails = renderInfo || {}`
Either way you can get rid of the if statement below though.

Reply:
Will sort